bindings/python/tv.cc

```diff
@@ -17,6 +18,16 @@ void tvFromJson(const std::vector<std::string>& input_json_per_rank_list, const
     // Load the configuration from serialized YAML
     YAML::Node viz_config = YAML::Load(input_yaml_params_str);
 
+    // Config Validator
+    ConfigValidator config_validator(viz_config);
+
+    // Check configuration
+    bool is_config_valid = config_validator.isValid();
+
+    // Throw error if configuration is invalid
+    if (!is_config_valid) {
+      throw std::runtime_error("The YAML configuration file is not valid: missing required paramaters: " + config_validator.getMissingRequiredParameters());
```

Comment:
To be consistent between error messages I suggest to add "vt-tv: " prefix and update the message as the following:
`"vt-tv: Error validating the configuration file: " + config_validator.getMissingRequiredParameters());"`